imagevector/images.yaml

```diff
@@ -72,6 +72,10 @@ images:
       confidentiality_requirement: 'low'
       integrity_requirement: 'low'
       availability_requirement: 'low'
+  - name: 'cloud.gardener.cnudie/responsibles'
+    value:
+    - type: 'githubUser'
+      username: 'shafeeqes'
```

Comment:
I'm not an expert in this area, but assigning a specific person for every image task seems inefficient. Shouldn't it be a team responsibility?

Reply:
Who is usually taking care for the CVEs in `nginx-ingress-controller`? Do you have dedicated people or it's a responibility for the current release responsible?

Reply:
OK, It doesn't really make sense to configure `cloud.gardener.cnudie/responsibles` for g/g components as you have your own schedule for release responsible who shall take care.